src/OneMightyRoar/PHP_ActiveRecord_Components/AbstractModel.php

```diff
@@ -560,13 +560,20 @@ public static function buildPagingOptions(array $paging_options, $page = null)
                 $order_desc = strcasecmp($order_desc, 'false') !== 0;
                 $order_dir = $order_desc ? 'DESC' : 'ASC';
             }
-
-            $order = $order_col . ' ' . $order_dir;
         } else {
-            // Add ticks around the first word (eg. $order = 'id DESC' becomes '`id` DESC')
-            $order = preg_replace('/(?<=\>)\b\w*\b|^\w*\b/', '`$0`', $order);
+            $order_tokens = explode(' ', $order);
```

Comment:
Hmm we could. That would mean the client did some weird input, but can't hurt here.